MapboxNavigation/NavigationView.swift

```diff
@@ -75,6 +75,34 @@ open class NavigationView: UIView {
     lazy var muteButton = FloatingButton.rounded(image: Images.volumeUp, selectedImage: Images.volumeOff)
     lazy var reportButton = FloatingButton.rounded(image: Images.feedback)
     
+    public enum FloatingButtonsPositions {
+        case topRight, topLeft
```

Comment:
The enumeration and its cases should each be documented to ensure that they appear in the jazzy docset.